OSRMTextInstructions.podspec

```diff
@@ -0,0 +1,52 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = "OSRMTextInstructions"
+  s.version = "0.0.1"
+  s.summary = "osrm-text-instructions parser for Swift and Objective-C."
```

Comment:
Developers perusing CocoaPods.org won’t have a clue what an osrm-text-instructions parser is.

> Transforms OSRM route reponses into human-readable instructions.